examples/yoti_example_django/yoti_example/views.py

```diff
@@ -24,11 +24,18 @@ class AuthView(TemplateView):
     def get(self, request, *args, **kwargs):
         client = Client(YOTI_CLIENT_SDK_ID, YOTI_KEY_FILE_PATH)
         activity_details = client.get_activity_details(request.GET['token'])
-        context = vars(activity_details.profile)
+        profile = activity_details.profile
+        profile_dict = vars(profile)
+
+        context = profile_dict.get('attributes')
         context['base64_selfie_uri'] = getattr(activity_details, 'base64_selfie_uri')
         context['user_id'] = getattr(activity_details, 'user_id')
 
-        selfie = context.get('profile', {}).get('selfie')
+        age_verified = profile.get_attribute('age_over:18')  # age condition defined in dashboard
```

Comment:
It just returns the None type if that is the case. It would be good to have a helper which would search for the `age_over`/`age_under` attributes, but this is just demonstrating it through the "get attribute by name" method at the moment